content/release/0.14.1.html

```diff
@@ -189,15 +189,15 @@ <h1>release 0.14.1 available</h1>
 <small>2007 Sep 4 </small>
 </div>
 <div class="col-md-4">
-   <p><a href="https://www.apache.org/dyn/closer.cgi/hadoop/common/hadoop-0.14.1/hadoop-0.14.1.tar.gz" class="btn btn-success">Download tar.gz</a></p>
+   <p><a href="https://archive.apache.org/dist/hadoop/common/hadoop-0.14.1/hadoop-0.14.1.tar.gz" class="btn btn-success">Download tar.gz</a></p>
    <p>
-     (<a href="https://dist.apache.org/repos/dist/release/hadoop/common/hadoop-0.14.1/hadoop-0.14.1.tar.gz.mds">checksum</a>
-     <a href="https://dist.apache.org/repos/dist/release/hadoop/common/hadoop-0.14.1/hadoop-0.14.1.tar.gz.asc">signature</a>)
+     (<a href="https://archive.apache.org/dist/hadoop/common/hadoop-0.14.1/hadoop-0.14.1.tar.gz.mds">checksum</a>
```

Comment:
this file doesn't exist. it looks like at the time we generated ".md5" files instead of a collections of hashes.

I assume this is true of many of the old releases. Could we run this list through a link checker? I'll try to find what tool we use over on HBase to automate this.